packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js

```diff
@@ -1,13 +1,10 @@
-const viewports = Terra.viewports('medium');
-
 // Color contrast will be resolved in https://github.com/cerner/terra-core/issues/1670
 const ignoredA11y = {
```

Comment:
not in scope of this pr, but is this disabled rule still necessary?

Reply:
I just removed it and tried. Receiving the following error: 
```
running chrome
AssertionError: expected no accessibility violations but got:
	{
  "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
  "help": "Elements must have sufficient color contrast",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=axeAPI",
  "id": "color-contrast",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-hour-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Hours</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeEmpty > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(2) > label[for=\"terra-time-hour-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-minute-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Minutes</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeEmpty > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(4) > label[for=\"terra-time-minute-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-hour-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Hours</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(2) > label[for=\"terra-time-hour-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-minute-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Minutes</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(4) > label[for=\"terra-time-minute-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-hour-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Hours</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#disabledDatetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(2) > label[for=\"terra-time-hour-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-minute-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Minutes</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#disabledDatetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(4) > label[for=\"terra-time-minute-input\"]"
      ]
    }
  ],
  "tags": [
    "cat.color",
    "wcag2aa",
    "wcag143"
  ]
}
    at Object.runAccessibilityTest (/Users/er047227/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessibility.js:23:35)
    at Context.<anonymous> (/Users/er047227/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/validate-element.js:77:28)
    at Promise (<anonymous>)
    at F (/Users/er047227/terra-framework/node_modules/babel-runtime/node_modules/core-js/library/modules/_export.js:36:28)
```

Reply:
Logged this issue: https://github.com/cerner/terra-framework/issues/744